hack/istio/install-bookinfo-demo.sh

```diff
@@ -148,10 +158,14 @@ if [ "${GATEWAY_YAML}" == "" ]; then
   GATEWAY_YAML="${ISTIO_DIR}/samples/bookinfo/networking/bookinfo-gateway.yaml"
 fi
 
+# If we are to delete, remove everything and exit immediately after
 if [ "${DELETE_BOOKINFO}" == "true" ]; then
   echo "====== UNINSTALLING ANY EXISTING BOOKINFO DEMO ====="
-  if [[ "$CLIENT_EXE" = *"oc" ]]; then
-    $CLIENT_EXE delete network-attachment-definition istio-cni -n ${NAMESPACE}
+  if [ "${IS_OPENSHIFT}" == "true" ]; then
+    if [ "${IS_MAISTRA}" != "true" ]; then
+      $CLIENT_EXE delete network-attachment-definition istio-cni -n ${NAMESPACE}
+    fi
+    $CLIENT_EXE delete security-context-constraints bookinfo-scc -n ${NAMESPACE}
```

Comment:
this is wrong, you get `error: the server doesn't have a resource type "security-context-constraints"`

change to `scc`
```suggestion
    $CLIENT_EXE delete scc bookinfo-scc -n ${NAMESPACE}
```

Reply:
I fixed this in my PR